### PR TITLE
use all things node 8 in blueprints

### LINKED ISSUE #8643: Addon blueprint using yarn fails tests on Travis
Addons generated with the `ember-cli@3.10.0` blueprint and using yarn fail on TravisCI because of a node version incompatibility.

The blueprint's `travis.yml` specifies `6` as the node version, but `ember-cli` [dropped node 6](https://github.com/ember-cli/ember-cli/commit/88a59e4363c6d51918454d40cf67d7c50f09e1f1) support in `3.10`, so when Travis runs `yarn` to install the addon dependencies, it fails with

```
$ yarn install --non-interactive
yarn install v1.16.0
[1/5] Validating package.json...
[2/5] Resolving packages...
[3/5] Fetching packages...
error ember-cli@3.10.0: The engine "node" is incompatible with this module. Expected version "8.* || >= 10.*". Got "6.17.1"
error Found incompatible module.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
The command "yarn install --non-interactive" failed and exited with 1 during .
Your build has been stopped.
```

`npm` (in node 6) apparently doesn't validate the `engines`, so this only happens when using `yarn`.

### PR DESCRIPTION
**Pull request by kellyselden**
I took the liberty of removing `npm config set spin false` because now that Travis runs `npm ci` instead of `npm test` by default, I think console noise is now longer an issue.

Should I update any other dependencies while I'm at it?

Closes #8643

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -48,9 +48,6 @@ jobs:
       after_success:
         - .travis/codecoverage.sh
 
-    - env: NAME=NODE_11
```
Comment (rwjblue) [inline .travis.yml]: We shouldn't remove this yet, Node 11 is still a supported platform (until 2019-06-01).

File: blueprints/app/files/package.json [hunk 2]
```diff
@@ -42,11 +42,11 @@
...
     "ember-welcome-page": "^4.0.0<% } %>",
     "eslint-plugin-ember": "^6.2.0",
-    "eslint-plugin-node": "^8.0.1",
+    "eslint-plugin-node": "^9.0.1",
     "loader.js": "^4.7.0",
     "qunit-dom": "^0.8.4"
   },
   "engines": {
-    "node": "6.* || 8.* || >= 10.*"
+    "node": "8.* || 10.* || >= 12.*"
```
Comment (rwjblue) [inline blueprints/app/files/package.json]: Node 11 is still supported until 2019-06-01:

```suggestion
    "node": "8.* || >= 10.*"
```

File: blueprints/module-unification-app/files/package.json [hunk 3]
```diff
@@ -42,11 +42,11 @@
...
     "ember-welcome-page": "^4.0.0<% } %>",
     "eslint-plugin-ember": "^6.2.0",
-    "eslint-plugin-node": "^8.0.1",
+    "eslint-plugin-node": "^9.0.1",
     "loader.js": "^4.7.0",
     "qunit-dom": "^0.8.4"
   },
   "engines": {
-    "node": "6.* || 8.* || >= 10.*"
+    "node": "8.* || 10.* || >= 12.*"
```
Comment (rwjblue) [inline blueprints/module-unification-app/files/package.json]:
```suggestion
    "node": "8.* || >= 10.*"
```

**Comment by kellyselden** [comment]
I was getting node 11 failures, but I'll try rolling that back.

File: blueprints/module-unification-app/files/package.json (hunk 3 above)
Comment (kellyselden) [reply blueprints/module-unification-app/files/package.json]: done

File: blueprints/app/files/package.json (hunk 2 above)
Comment (kellyselden) [reply blueprints/app/files/package.json]: done

File: .travis.yml (hunk 1 above)
Comment (kellyselden) [reply .travis.yml]: done

**Pull request merged**